.github/workflows/preview.yml

```
@@ -0,0 +1,102 @@
+name: Preview
+
+on:
+  pull_request:
+
+jobs:
+  preview:
+    name: Run preview
+    runs-on: ubuntu-latest
+    env:
+      PREVIEW_HOSTNAME: ep-preview.click
```

Could we have this as an environment variable please? Makes it reusable for others.
```suggestion
      PREVIEW_HOSTNAME: ${{ vars.PREVIEW_HOSTNAME }}
```

---

I'm going to keep it as-is for now, because there are other things in that GHA that are enviornment-specific, so might be better to just have a separate commit/PR dealing with all those changes at once.  – for example there's a separate deployment hostname (static.ep).
Separately it's probably a good idea to structure it as separate build/preview/deploy steps, so could be done as part of that change. Currently deploy/preview share a lot of logic that can be put in a separate "build" step.